### ISSUE TITLE
Convert reviewer sharing names CSV to an importable CSV

### ISSUE THREAD

**Issue by de-code** [elifesciences/data-pipeline#78]
Depends on #77 

Create a CSV `<time>_manuscript_version_reviewer_sharing_name.csv` with the following columns (in addition to the standard columns):

* _msid_ (extract from _Manuscript Tracking No._)
* _version_position_in_ms_ (extract from _Manuscript Tracking No._? TODO review)
* _reviewer_person_id_ (from _Reviewer Person ID_)
* _share_name_ (from _Yes (1) / No (2)_, suggest to map as boolean 1 => 1, 2 => 0, 0 => _blank_)

As is the case with all those CSV exports, they first three rows contain meta data that can be ignored. Although in this case we could get the timestamp from the second row (e.g. `Generated on July 9, 2018`).

**Comment by de-code** [comment]
@seanwiseman do you think it will be okay to extract the _version_position_in_ms_ from the long _manuscript-number_? e.g.

* `*-12345`: version_position_in_ms=0
* `*-12345R1`: version_position_in_ms=1
* `*-12345R2`: version_position_in_ms=2

Otherwise we could try to extract more information via SQL (which generates the CSV).

**Comment by seanwiseman** [comment]
> (in addition to the standard columns)

@de-code could you just confirm what these are as this is not an `XML` file?

So far we have:
- `msid` (extract from Manuscript Tracking No.)
- `version_position_in_ms` (extract from Manuscript Tracking No.? TODO review)
- `reviewer_person_id` (from Reviewer Person ID)
- `share_name` (from Yes (1) / No (2), suggest to map as boolean 1 => 1, 2 => 0, 0 => blank)
...

**Comment by de-code** [comment]
I guess we still need:

* create_date
* zip_name
* xml_file_name

Maybe we should rename them. I would imagine that every CSV that we present to the db-manager should have the same set of 'source' columns, independent of where the data is from. Otherwise it would have to know that the file came from a different kind of source. Would you agree?

**Comment by seanwiseman** [comment]
Yes I agree.

- `create_date` = fine as is
- `zip_name` = not sure about this, allow blank?
- `xml_file_name` = `source_file_name`?

**Comment by de-code** [comment]
Looks like currently we are not using `zip_name` in `db-manager` except for `dimPerson` and `dimPersonRole` where it becomes `source_file_name` in the dimension table (`"$zip_name/$xml_file_name"`). So perhaps drop `zip_name` and rename `xml_file_name` like you suggested. And were the XML is coming from a zip we concatenate the name in the same way.

Probably best as a separate issue?
